memcached/src/play/cache/MemcachedImpl.java

```diff
@@ -40,6 +41,14 @@ public MemcachedClient getClient() {
     return client;
   }
 
+  @Override
+  public CacheImpl instance(@Nonnull Properties playProperties) throws IOException {
+    if (uniqueInstance == null) {
+      uniqueInstance = new MemcachedImpl(playProperties);
+    }
+    return null;
+  }
```

Comment:
I don't understand this place.
1. non-static method in `MemcachedImpl` returns another instance of `MemcachedImpl`? 
2. This method always returns `null`.

Reply:
As mentioned in the other review-comment-thread, the code has been greatly improved now. Also due to this comment.